Makefile

```diff
@@ -152,7 +152,9 @@ functions-examples-all:
 	done
 
 test-go-mod:
-	./hack/for-each-module.sh "go list -m -json all > /dev/null && go mod tidy -v"
+	./hack/for-each-module.sh $$(pwd)/hack/replace.sh; \
```

Comment:
I tried using `$$KUSTOMIZE_ROOT` instead of `$$(pwd)`, but couldn't figure out how to use the exported `hack/for-each-module.sh` variable here.